### ISSUE TITLE
No webchat popup

### ISSUE THREAD

**Issue by xnox** [canonical/ubuntu.com#12326]
Ubuntu Pro page doesn't have webchat pop-up to discuss Pro

---
*Reported from: https://ubuntu.com/pro*

**Comment by sowasred2012** [comment]
@lechsandecki is this something you want on the /pro page?

**Comment by sowasred2012** [comment]
@jpmartinspt could you raise this with Lech pls?

**Comment by lechsandecki** [comment]
yes, i think it would be nice to have a pop-up there, thanks for raising that!

**Comment by jpmartinspt** [comment]
The popup has been re-enabled.
